x-pack/test/functional/page_objects/reporting_page.js

```diff
@@ -169,6 +169,12 @@ export function ReportingPageProvider({ getService, getPageObjects }) {
       const toTime = 'Sep 23, 1999 @ 18:31:44.000';
       await PageObjects.timePicker.setAbsoluteRange(fromTime, toTime);
     }
+
+    async canReportBeCreated() {
```

Comment:
It is never used in test. Could you comment?

Reply:
yes, removed.